.circleci/config.yml

```diff
@@ -237,6 +237,32 @@ jobs:
 
             docker push codacy/$CIRCLE_PROJECT_REPONAME:$PUBLISH_VERSION
 
+  publish_dockerhub_stable:
```

Comment:
Duplicating all of this seems a bit too much. Can't we do something like the integration tests?
You only need to check that you cannot use env variables in other variables but you can do the same trick.